time stamp

float64 cpu_usage
float64[] percpu_usage
float64 cpu_frequency
float64[] percpu_frequency
float64[] percore_temp
float64 ram_usage
float64 disk_usage
